packages/material-ui/src/Button/Button.js

```diff
@@ -3,10 +3,10 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import { chainPropTypes } from '@material-ui/utils';
```

Comment:
Unless we revisit #13391 this will pull in the complete `@material-ui/utils` package since `Button.js` is currently transpiled to commonJS modules. This is not an issue for a prop validator since those will be removed in prod anyway due to `babel-plugin-remove-prop-types` as far as I know. Just wanted to caution against named imports for now.